documentation/dev-portal/src/community-resources/community-applications-and-guides.md

```diff
@@ -55,7 +55,53 @@ We love seeing our developer community create applications using Nym. If you wou
 
 <!--NymDrive *end-->
 
+<!--NymDashboard *start-->
 
+## <img src='../images/profile_picture/nym_dashboard_pp.svg' style="float: right; width: 75px; height: 75px;">Nym Dashboard
 
+> Developed by No Trust Verify, this dashboard is a great tool to get information about the mixnet, gateways and mixnodes.
+* [Deployed Website App](https://status.notrustverify.ch/d/CW3L7dVVk/nym-mixnet?orgId=1)
 
+<!--NymDashboard *end-->
+
+<!--IsNymUp *start-->
+ 
+## <img src='../images/profile_picture/pastenym_ntv_pp.png' style="float: right; width: 75px; height: 75px;">Is Nym Up
+ 
+> A tool that helps check the heath of the Nym network as well as some mixnet related statistics.
+* [Deployed Website App](https://isnymup.com/)
+
+<!--IsNymUp *end-->
+
+
+<!--DarkFi *start-->
+
+## <img src='../images/profile_picture/darkfi_over_nym_pp.png' style="float: right; width: 75px; height: 75px;">DarkFi over Nym
+
+> In this example, DarkFi used Nym's Mixnet to allow users to anonymously connect to other peers in the Darkfi network.
```

Comment:
This needs to be more specific IMO - DarkFi can now use the mixnet as a pluggable transport for apps such as `ircd`, their p2p version of `irc`

Reply:
resolved this